### PR TITLE
[14.0][FIX] project_timeline_hr_timesheet: Set the correct class in all cases

### PR DESCRIPTION
**Pull request by victoralmau**
Set the correct class in all cases.
ejemplo

Please @pedrobaeza and  @CarlosRoca13 can you review it?

@Tecnativa TT40299

### CODE REVIEW

File: project_timeline_hr_timesheet/views/project_task_view.xml [hunk 1]
```diff
@@ -14,7 +14,13 @@
             <xpath expr="//small[@name='planned_hours']" position="after">
                 <div class="o_project_timeline_hr_timesheet_progress">
                     <div
-                        t-att-class="record.remaining_hours &lt; 0 and 'danger' or ''"
+                        t-if="record.remaining_hours &lt; 0"
```
Comment (pedrobaeza) [inline project_timeline_hr_timesheet/views/project_task_view.xml]: I don't think you need this, just put the other as `t-attf-class`

Comment (victoralmau) [reply project_timeline_hr_timesheet/views/project_task_view.xml]: I have tried to make that change but it does not work in this case.
ejemplo-1

Comment (pedrobaeza) [reply project_timeline_hr_timesheet/views/project_task_view.xml]: It's weird, as the Qweb should do it:

https://www.odoo.com/documentation/14.0/es/developer/reference/javascript/qweb.html

Maybe it's not being rendered properly? Anyway, if not possible, just drop a comment here to know why it's done that way. Please always remember to document each weird thing for not questioning each time why is there.

Comment (victoralmau) [reply project_timeline_hr_timesheet/views/project_task_view.xml]: I finally found out how to do it in one line.

**Review by CarlosRoca13 — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Pull request merged**